src/services/orderServices/updateOrderService.ts

```diff
@@ -0,0 +1,127 @@
+import { Request, Response } from 'express';
+import { getManager, EntityManager, Repository } from 'typeorm';
+import { Order } from '../../entities/Order';
+import { Product } from '../../entities/Product';
+import { User } from '../../entities/User';
+import { OrderItem } from '../../entities/OrderItem';
+import { Transaction } from '../../entities/transaction';
+import { responseError, sendErrorResponse, sendSuccessResponse } from '../../utils/response.utils';
+import sendMail from '../../utils/sendOrderMail';
+
+export const updateOrderService = async (req: Request, res: Response) => {
+    const { orderId } = req.params;
+    const { orderStatus } = req.body;
```

Comment:
@UwicyezaG , @elijahladdie Can you check if provided orderStatus is among the ones defined as possible value in "orderStatus" field